babel/messages/frontend.py

```diff
@@ -248,12 +248,12 @@ class extract_messages(Command):
          'set project version in output'),
         ('add-comments=', 'c',
          'place comment block with TAG (or those preceding keyword lines) in '
-         'output file. Separate multiple TAGs with commas(,)'),
+         'output file. Separate multiple TAGs with commas(,)'),  # TODO: Support repetition of this argument
```

Comment:
Not strictly necessary, but I'm a fan of linking these to issue numbers, like TODO(#386): or something.